src/ucp/wireup/wireup_cm.c

```diff
@@ -478,8 +531,12 @@ static void ucp_cm_disconnect_cb(uct_ep_h uct_cm_ep, void *arg)
     uct_worker_cb_id_t prog_id = UCS_CALLBACKQ_ID_NULL;
     ucp_cm_disconnect_progress_arg_t *progress_arg;
 
+    ucs_debug("ep %p: CM remote disconnect callback invoked, flags 0x%x",
+              ucp_ep, ucp_ep->flags);
+
     progress_arg = ucs_malloc(sizeof(*progress_arg), "disconnect_progress_arg");
     if (progress_arg == NULL) {
+        ucs_error("failed to allocate ucp_cm_disconnect_progress_arg");
```

Comment:
why do we even need progress_arg, pass ucp_ep as the callback arg
we have only one cm now anyway

Reply:
the reason is not number of CMs but number of CM lanes, we have plans for multi lanes in near future.. OK, will remove for now since there are > 1 way to implement this.